tasks/holesky/001-pectra-defense/VALIDATION.md

```diff
@@ -0,0 +1,121 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff
+are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state
+  changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain
+  Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## Nested Safe State Overrides and Changes
+
+This task is executed by the nested 2/2 `ProxyAdminOwner` Safe. Refer to the
+[generic nested Safe execution validation document](../../../NESTED-VALIDATION.md)
+for the expected state overrides and changes.
+
+The `approvedHashes` mapping **key** of the `ProxyAdminOwner` that should change during the simulation is
+- Council simulation: `0x3394b41f3c05f4c1bb2767a59f18fd9cc4497d7a8781dca9206396044c5d4a80`
+- Foundation simulation: `0xfd5f9ae7ec744b3e9bbfd555849eb2f5727c46fe3e5cd532c7e102c0934e99c1`
+
+calculated as explained in the nested validation doc.
+
+Additionally, the nonces [will increment by one](../../../NESTED-VALIDATION.md#nonce-increments).
+
+## State Changes
+
+Note: The changes listed below do not include safe nonce updates or liveness guard related changes. Refer to the
+[generic nested Safe execution validation document](../../../NESTED-VALIDATION.md)
+
+### `0xF1408Ef0c263F8c42CefCc59146f90890615A191` (`DisputeGameFactoryProxy`)
+
+- **Key**: `0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f9e` <br/>
+  **Before**: `0x000000000000000000000000c0b6c36e14755296476d5d8343654706fe0978a8` <br/>
+  **After**: `0x000000000000000000000000d3f6b7df02d5617c7969c1e7a005170a2633c895` <br/>
+  **Meaning**: Updates the implementation for game type 1. Verify that the new implementation is set using
+  `cast call 0xF1408Ef0c263F8c42CefCc59146f90890615A191 "gameImpls(uint32)(address)" 1`.
```

Comment:
Seems better to show `cast index` here to compute the slot number? Since when signing, you can't verify the new implementation is set using that command. But we can verify the old implementation, so that's another option
```suggestion
  **Meaning**: Updates the implementation for game type 1. Verify that the old implementation is set in this slot using
  `cast call 0xF1408Ef0c263F8c42CefCc59146f90890615A191 "gameImpls(uint32)(address)" 1`.
```